### PR TITLE
fix(utilities): correct offset for scroll to clause - I79

### PR DESCRIPTION
**Pull request by archhhh**
# Issue #79

### Changes
- change in navigateClause.js,  passing different parameters to scrollIntoView function

### Related Issues
- Issue # 79

### CODE REVIEW

**Review by jolanglinais — CHANGES_REQUESTED** [review]
Thanks for the PR! Were you able to test this and see it working? I recorded a Loom, it looks like there are still bugs:

https://www.loom.com/share/c97d5f7eeb944db39654cad53b030ec2

Also, you'll want to add a DCO (Developers Certificate of Originality) per [the instructions](https://github.com/accordproject/template-studio-v2/blob/master/CONTRIBUTING.md):
> Ensure you provide a DCO sign-off for your commits using the -s option of git commit. For more information see [how this works](https://github.com/probot/dco#how-it-works).

**Comment by archhhh** [comment]
I added new commits with changes, tested code and it seems to be working. Also added DCO sign-off to the last commit.

**Comment by jolanglinais** [comment]
@archhhh this is looking really rad! I'll have more of a chance to look it over tonight or tomorrow - also hoping to get @DianaLease's feedback.

Also, I think you'll need to amend your commits with the signoff and force push them.

**Comment by archhhh** [comment]
I think I signed off everything, could you please recheck it?

File: src/utilities/navigateClause.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as R from 'ramda';
 
 import store from '../store';
+import { min } from 'moment-mini';
```
Comment (DianaLease) [inline src/utilities/navigateClause.js]: do we need this extra library? doesn't look like you are actually using it.

**Review by DianaLease — CHANGES_REQUESTED** [review]

File: src/utilities/navigateClause.js (hunk 1 above)
Comment (jolanglinais) [reply src/utilities/navigateClause.js]: I think we are here: https://github.com/accordproject/template-studio-v2/pull/104/files#diff-16f57a2341a65f078824cec5da6ef4e8R70

@archhhh why is this library used?

Comment (archhhh) [reply src/utilities/navigateClause.js]: Yes, no need to use it, I removed that library in the last commit.

File: src/utilities/navigateClause.js [hunk 2]
```diff
@@ -45,13 +45,55 @@ const findClauseNode = (clauseId) => {
...
+    return node;
+  }
+
+  return getScrollParent(node.parentNode) || document.body;
+}
+
+let animationFrame;
+
+const scrollTo = (element, value) => {
+  console.log(`${element.scrollTop} ${value}`);
```
Comment (jolanglinais) [inline src/utilities/navigateClause.js]: Can we remove this `console.log`?

File: src/utilities/navigateClause.js [hunk 3]
```diff
@@ -45,13 +45,55 @@ const findClauseNode = (clauseId) => {
...
+  if(Math.abs(element.scrollTop - value) > 2){
+    let oldValue = element.scrollTop;
```
Comment (jolanglinais) [inline src/utilities/navigateClause.js]: It looks like `oldValue` can be defined with `const` since we don't need to reassign it.

**Review by jolanglinais — CHANGES_REQUESTED** [review]
I have a few changes.

**Comment by archhhh** [comment]
Added new commit with requested changes.

**Review by DianaLease — APPROVED** [review]
This looks good to me!

**Review by jolanglinais — APPROVED** [review]

**Comment by jolanglinais** [comment]
Super cool @archhhh, thanks for the contribution!

**Pull request merged**